package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "github.com/livekit/protocol",
+  "private": true,
+  "version": "1.10.1",
```

Comment:
Would we need to update the version here when tagging a new release?

Same question for `packages/javascript/package.json`.

What's the difference between the two, btw? It's for compatibility, I assume?

Reply:
No, this will automatically be done by the changesets PR. 
[Here's](https://github.com/livekit/client-sdk-js/pull/1066/files) an example of what that looks like on the client-sdk-js. 
Once merged, the release action will create the git tag. 

One (the one on root) is for tracking the changes of the JS package, the other one (in root) will track the golang module.